addons/project/data/mail_channel_demo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="mail_channel_project_task" model="mail.channel">
+            <field name="name">Projects &amp; Tasks</field>
+            <field name="group_public_id" ref="group_project_user"/>
+            <field name="group_ids" eval="[(4, ref('group_project_user'))]"/>
+        </record>
+        <function model="project.project" name="message_subscribe"
```

Comment:
1. Using demo record ```<record...>```, whenever this file is installed second time on same db, it will raise unique constraint violation error on channel ```unique(res_model,res_id,channel_id)```
2. ```message_subscribe``` will add given channel and subtype to follower of this project